CWE/events/italy.txt

```diff
@@ -54,460 +99,232 @@ country_event = {
-country_event = {
-  id = 8007741
-  title = EVT_8007741_NAME
-  desc = EVT_8007741_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes
-
-	option = {
-		name = EVT_8007741_A
-		any_pop = { militancy = 1.0 consciousness = 1 }
-		set_country_flag  = very_radical_ICP
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.02 value = communist }
-		}
-	}
-  
-}
-
-#Dispersion of Italian Communist Party
-country_event = {
-  id = 8007742
-  title = EVT_8007742_NAME
-  desc = EVT_8007742_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes
-
-	option = {
-		name = EVT_8007742_A
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = -0.10 value = communist }
-		}
-	}
 	
 }
 
-#Italian Communist Party at helm
+#The Special Laws
 country_event = {
-  id = 8007743
-  title = EVT_8007743_NAME
-  desc = EVT_8007743_DESC
-  picture = "nwo2_italian_communist_party"
+	id = 8007772
+	title = "The $YEAR$ Special Laws"
+	desc = EVTDESC8007772
+	picture = "italian_parliament"
   
 	fire_only_once = yes
-  
+
 	trigger = {
 		tag = ITA
-		has_global_flag = blocsenabled
-		ruling_party_ideology = communist
-		government = proletarian_dictatorship
-		election = no
-	}
-
-	option = {
-		name = EVT_8007743_A
-		
-		#Break alliances with non communists
-		any_country = {
-			limit = { NOT = { ruling_party_ideology = communist } }
-			leave_alliance = THIS
-			relation = { who = THIS value = -200 }
-		}
-		
-		#Be friends with fellow communists
-		any_country = {
-			limit = { ruling_party_ideology = communist }
-			relation = { who = THIS value = 200 }
-		}
+		year = 1948
+		NOT = { year = 1990 }
 		
-		#Citizens become more communist
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
+		government = democracy
+		is_vassal = no
+		war = no
 		
-		#USA choice to declare war
-		USA = { country_event = 8007744 }
+		AND = { has_country_flag = 1948_ITA_FDP has_country_flag = ITA_Togliatti_shooting }
+		NOT = { has_country_flag = 1948_ITA_Centrism }
 	}
 
-}
-
-#Hardline communists gain power in Italy
-country_event = {
-  id = 8007744
-  title = EVT_8007744_NAME
-  desc = EVT_8007744_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes # USA
-
-	option = {
-		name = "It was the choice of the Italian people"
-		RUS = { country_event = 8007745 } #Victory for the Russians
-		ai_chance = { factor = 0.6 }
-	}
+	mean_time_to_happen = { months = 4 }
 
 	option = {
-		name = "We have to intervene!"
-		
-		war = { 
-			target = ITA 
-			attacker_goal = { casus_belli = install_democracy } 
-			defender_goal = { casus_belli = humiliate } 
-		}
-		
-		#USSR choice to defend
-		random_country = {
-			limit = { tag = RUS government = proletarian_dictatorship }
-			country_event = 8007745 #RUS choice to ally with ITA
-		}
+		name = "Enact the new laws!"
+		relation = { who = USA value = -100 }
```

Comment:
Please use a random_country scope to affect the relations, following the example below.

random_country = {
 			limit = { tag = USA government = democracy is_vassal = no }
 			relation = { who = THIS value = -100 }
 		}